v2/spanner-common/terraform/samples/README.md

```diff
@@ -0,0 +1,53 @@
+## Terraform samples for common spanner setups
+
+This repository provides samples for scenarios that emulate common customer
+scenarios. These are especially useful is setting up test infrastructure that
+resembles
+a customers' environment, in order to run various Spanner migration tooling such
+as -
+
+1. [SMT](https://github.com/GoogleCloudPlatform/spanner-migration-tool)
```

Comment:
Updated the SMT link to point to - https://googlecloudplatform.github.io/spanner-migration-tool/
Reverse and Live replication URLs would remain the same since we want this to be modular by design going forward. SMT documentation being the source of truth for live, bulk and reverse made sense when SMT was the starting point for all of these products, which we don't want to do.